src/sagemaker/jumpstart/types.py

```diff
@@ -1707,3 +1746,37 @@ def __init__(
         self.nearest_model_name = nearest_model_name
         self.data_input_configuration = data_input_configuration
         self.skip_model_validation = skip_model_validation
+
+
+class HubArnExtractedInfo(JumpStartDataHolderType):
```

Comment:
why not use `@dataclass`?

Reply:
That's what I had initially, but Evan recommended `JumpStartDataHolderType` instead since it performs much of the same capabilities. I'm indifferent